napalm/base/interface_map.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+interface_map = {
+  "base_interfaces": {
```

Comment:
Why a nested dictionary as opposed to two separate dictionaries? I think they should be two separate dictionaries.